spec/rubocop/cop/rspec/cop_spec.rb

```diff
@@ -21,13 +21,15 @@
 
   let(:fake_cop) do
     stub_const('RuboCop::RSpec', Module.new)
-    # rubocop:disable ClassAndModuleChildren, RSpec/LeakyConstantDeclaration
+    # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
Do you think it's necessary to put `disable` statements on separate lines?

Reply:
Yes it was tripping up on line length 😬